### PR TITLE
Framework update

### PR DESCRIPTION
**Pull request by darosior**

### CODE REVIEW

**Comment by darosior** [comment]
Dunno why Github keeps showing the commits of the already-merged 'rc 0.3' PR

**Comment by darosior** [comment]
Ok, fixed Github's glitch.

File: aquarium.py [hunk 1]
```diff
@@ -30,9 +32,9 @@
 REVAULTD_SRC_DIR = os.path.join(SRC_DIR, "revaultd")
 REVAULT_GUI_SRC_DIR = os.path.join(SRC_DIR, "revault-gui")
 SHELL = os.getenv("SHELL", "bash")
-COORDINATORD_VERSION = os.getenv("COORDINATORD_VERSION", "9b90fcd3ad5dfb51bf50dcaef859c5cc00dbfe3a")
-COSIGNERD_VERSION = os.getenv("COSIGNERD_VERSION", "e41e5cafc33568f61076700ba360f831e3b072cc")
-REVAULTD_VERSION = os.getenv("REVAULTD_VERSION", "gethistory-2")
+COORDINATORD_VERSION = os.getenv("COORDINATORD_VERSION", "master")
+COSIGNERD_VERSION = os.getenv("COSIGNERD_VERSION", "master")
+REVAULTD_VERSION = os.getenv("REVAULTD_VERSION", "master")
 REVAULT_GUI_VERSION = os.getenv("REVAULT_GUI_VERSION", "gethistory")
```
Comment (edouardparis) [inline aquarium.py:38]: you can update on `revault:master` now

Comment (darosior) [reply aquarium.py:38]: Let's do that in a followup PR to not have to rebase everything

**Pull request merged**